modules/openid_auth/README_V2.yml

```diff
@@ -0,0 +1,147 @@
+openapi: '3.0.0'
+info:
+  version: 0.0.1
+  title: Validation
+  description: |
+    Oauth Validation for internal 3rd Party Services
+
+    ## Background
+
+
+
+    ## Design
+
+    ### Authorization
+
+    API requests are authorized using a Bearer token issued through an OpenID
+    Connect service to allow third-party applications. The token should be
+    submitted as an `Authorization` header in the form `Bearer <token>`.
+
+    ### Token Validation Request
+
+    Allows a third-party application to request the validated token and VA Identifier ICN for an individual:
+
+    1. Client Request: POST https://api.va.gov/internal/auth/v2/validation
+       * Provide the Bearer token as a header: `Authorization: Bearer <token>`
+       * Provide a body parameter for appropriate audience for validation
+         aud: https://api.va.gov/services/fhir
+
+    2. Status Response: A JSON API object with the individual's validated token and VA identifier ICN
```

Comment:
V2 returns more than just ICN... worth mentioning that here?

Reply:
Good catch.  I had updated the schema but not this blurb.

Due to lines of code restraints, I've broken out this documentation route to a separate PR.
https://github.com/department-of-veterans-affairs/vets-api/pull/6363